contentcuration/contentcuration/static/js/edit_channel/sharedComponents/ProgressOverlay.vue

```diff
@@ -74,6 +75,7 @@
       defaultErrorText:
         'An unexpected error has occurred. Please try again, and if you continue ' +
         'to see this message, please contact support via the Help menu.',
+      finishedMessage: 'Operation complete! Click "Close" to update the page.',
```

Comment:
Not a blocker, but should we rename the "Close" button to say something like "Refresh"? That way, if things get really slow, people know that it's reloading, not the close operation is broken?